ignite/contrib/handlers/param_scheduler.py

```diff
@@ -581,14 +592,25 @@ def simulate_values(cls, num_events, schedulers, durations, param_names=None, **
                     "Argument param_names should be list or tuple of strings, but given {}".format(param_names)
                 )
 
+        param_schedulers = [s for s in schedulers if isinstance(s, ParamScheduler)]
+        param_group_schedulers = [s for s in schedulers if isinstance(s, ParamGroupScheduler)]
+
+        param_optimizers = [s.optimizer for s in param_schedulers]
+        param_group_optimizers = [s.optimizer for sc in param_group_schedulers for s in sc.schedulers]
+        optimizer = list(set(param_optimizers + param_group_optimizers))
+        if len(optimizer) != 1:
+            raise ValueError("schedulers should be related to same optimizer")
```

Comment:
@sdesrozis can you please add a test for that too ?

Reply:
Yes !